tests/e2e/vmlistener_test.go

```diff
@@ -16,6 +16,8 @@
 // In case of vm/esx host restart vDVS cleans up stale attachment details
 // the volume.
 
+// +build unstable
```

Comment:
There is not any straight forward way that I know of. Selective run is possible through `go test -run TestNameRegexp` .. we need to come up with the `TestNameRegexp` such that we can exclude certain tests. 

There are other alternative that go test community user recommends like writing interceptor that filter out tests.